# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/nsFrameIterator.h
# Commit: 8bf69c3fd814
# Full Hash: 8bf69c3fd81419337cb93f96020bb5e88a333d32
# Author: Jan Varga <Jan.Varga@gmail.com>
# Date: 2012-08-07 13:39:24
# Regressor Bug: 779684
# File Overlap Count: 1
# Description:
#   Bug 779684 - Move FrameIteratorFlags to nsFrameIterator class. r=roc
# ==============================================================================

diff -r 78aaaf0d8185 -r 8bf69c3fd814 layout/base/nsFrameIterator.h
--- a/layout/base/nsFrameIterator.h	Mon Aug 06 14:51:33 2012 -0600
+++ b/layout/base/nsFrameIterator.h	Mon Aug 06 20:27:04 2012 -0400
@@ -16,13 +16,6 @@
   ePostOrder
 };
 
-enum FrameIteratorFlags {
-  FLAG_NONE = 0,
-  FLAG_LOCK_SCROLL = 1 << 1,
-  FLAG_FOLLOW_OUT_OF_FLOW = 1 << 2,
-  FLAG_VISUAL = 1 << 3
-};
-
 class nsFrameIterator
 {
 public:
@@ -38,6 +31,13 @@
 
   void Last();
   void Prev();
+
+  enum FrameIteratorFlags {
+    FLAG_NONE = 0,
+    FLAG_LOCK_SCROLL = 1 << 1,
+    FLAG_FOLLOW_OUT_OF_FLOW = 1 << 2,
+    FLAG_VISUAL = 1 << 3
+  };
 protected:
   void      setCurrent(nsIFrame *aFrame){mCurrent = aFrame;}
   nsIFrame *getCurrent(){return mCurrent;}